src/taipy/config/_config_comparator/_config_comparator.py

```diff
@@ -107,7 +107,7 @@ def __log_message(
             self.__logger.error(
                 f"The {old_config_str} is conflicted with the {new_config_str}:\n\t" + "\n\t".join(conflicted_messages)
             )
-            self.__logger.error("To override these changes, run your application with --force option.")
+            self.__logger.error("To override these changes, run your application with --override option.")
```

Comment:
I don't like the name override. 

what about `--force-run`?

"To force running the application with the changes, please run your application with --force-run option."

Reply:
I don't have a strong opinion on this. Can you leave a comment on the Back-end Discussion thread so others (FloJ maybe) can vote on this?